wfe/wfe.go

```diff
@@ -92,6 +92,16 @@ const (
 	// http://www.itu.int/rec/T-REC-X.509-201210-I/en
 	unusedRevocationReason       = 7
 	aACompromiseRevocationReason = 10
+
+	// authzReuseEnvVar defines an environment variable name used to provide a
+	// percentage value for how often Pebble should try to reuse valid authorizations
+	// for each identifier in an order. The percentage is independent of whether a
+	// valid authorization exists or not for each identifier in an order.
+	authzReuseEnvVar = "PEBBLE_WFE_AUTHZREUSE"
+
+	// The default value when PEBBLE_WFE_AUTHZREUSE is not set, how often to try
+	// and reuse valid authorizations?
```

Comment:
small nit: the trailing `?` reads to me as uncertainty :-)
```suggestion
	// and reuse valid authorizations.
```